crates/matrix-sdk/src/event_cache/store.rs

```diff
@@ -38,6 +43,7 @@ impl Default for RoomEvents {
 }
 
 impl RoomEvents {
+    /// Build a new `Self` with zero events.
```

Comment:
Addressing the current type as `Self` in documentation continues to be quite jarring.

```suggestion
    /// Build a new [`RoomEvents`] struct with zero events.
```